eslintrc.json

```diff
@@ -1,169 +1,156 @@
 {
-  "ecmaFeatures": {
-    "modules": true
-  },
+  "parser": "babel-eslint",
 
   "env": {
-    "browser": true,
-    "es6": true,
-    "node": true
-  },
-
-  "globals": {
-    "document": false,
-    "escape": false,
-    "navigator": false,
-    "unescape": false,
-    "window": false
+    "commonjs": true,
```

Comment:
I think the only global it exposes that we *might* want is `require`, but since we are trying to use ES6 modules... it's not really necessary. [commonjs globals](https://github.com/sindresorhus/globals/blob/master/globals.json#L844)